.github/workflows/github_release.yaml

```diff
@@ -0,0 +1,133 @@
+name: Release to GitHub
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  wf_vars:
+    name: "Set Workflow Variables"
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ env.VERSION }}
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Set Base Environment Variables
+        run: |
+          echo >> $GITHUB_ENV VERSION="v-$(date +%Y.%m.%d-%H%M%S)"
+
+  build_router_gh_release:
+    name: "GitHub Release: Build Router"
+    needs:
+      - wf_vars
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: prime-router
+    env:
+      # These are for CI and not credentials of any system
+      POSTGRES_USER: prime
+      POSTGRES_PASSWORD: changeIT!
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+      - name: Set Environment Variables
+        run: |
+          echo >> $GITHUB_ENV DB_USER=${POSTGRES_USER}
+          echo >> $GITHUB_ENV DB_PASSWORD=${POSTGRES_PASSWORD}
+      - name: Build Router Package
+        run: bash ./build.sh -- gradle clean package
+        # Archive artifact
+      - name: 'Tar router files'
+        run: tar -czvf prime-router-build.tar.gz build
+      - name: Upload Router Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: prime-router-build-${{ needs.wf_vars.outputs.version }}
+          path: prime-router/prime-router-build.tar.gz
+          retention-days: 1
+
+  build_frontend_react_gh_release:
+    name: "GitHub Release: Build Frontend"
+    needs:
+      - wf_vars
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: frontend-react
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }} yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - run: yarn install
+      - name: Build Frontend
+        run: yarn run build
+        # Archive artifact
+      - name: 'Tar frontend files'
+        run: tar -czvf static-website-react.tar.gz build
+      - name: Upload Frontend Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: static-website-react-${{ needs.wf_vars.outputs.version }}
+          path: frontend-react/static-website-react.tar.gz
+          retention-days: 1
+
+  publish_github_release:
+    name: "Publish to Github"
+    needs:
+      - wf_vars
+      - build_router_gh_release
+      - build_frontend_react_gh_release
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out changes"
```

Comment:
Nit: Can you make the quotes and capitalization in the "name" field consistent?

Reply:
Fixed